swap/src/asb/command.rs

```diff
@@ -52,6 +53,42 @@ pub enum Command {
         about = "Prints the Bitcoin and Monero balance. Requires the Monero Wallet RPC to be running."
     )]
     Balance,
+    #[structopt(about = "Contains sub-commands for recovering swap manually.")]
+    ManualRecovery(ManualRecovery),
+}
+
+#[derive(structopt::StructOpt, Debug)]
+pub enum ManualRecovery {
+    #[structopt(
+        about = "Cancel requires the cancel timelock to be expired. Cancel is a requirement for refund and punish."
```

Comment:
These are power user commands. I am not sure how much we want to go into detail here.
We could opt for using [`Arg::long_help`](https://docs.rs/structopt/0.3.21/structopt/#help-messages) but I am not sure if all users will get this. 
I was planning to elaborate more on this in the ASB docs instead of adding really long help texts to the commands. 

Someone who wants to properly use these commands HAS to have a proper understanding of the protocol.
I am not sure how much more education I can and want to pack into the help of these commands. 

I can add a sentence along the lines of `Publishes the Bitcoin cancel transaction.` - but I expect a user to expect that because they understand the protocol prior to using these commands. Do you have some concrete ideas as in what to add?

Reply:
> These are power user commands. I am not sure how much we want to go into detail here.
> We could opt for using [`Arg::long_help`](https://docs.rs/structopt/0.3.21/structopt/#help-messages) but I am not sure if all users will get this.
> I was planning to elaborate more on this in the ASB docs instead of adding really long help texts to the commands.
> 
> Someone who wants to properly use these commands HAS to have a proper understanding of the protocol.
> I am not sure how much more education I can and want to pack into the help of these commands.
> 
> I can add a sentence along the lines of `Publishes the Bitcoin cancel transaction.` - but I expect a user to expect that because they understand the protocol prior to using these commands. Do you have some concrete ideas as in what to add?

Yeah something along those lines just so that it reads better in isolation:

`Publishes the Bitcoin cancel transaction. By default, the cancel timelock will be enforced. A confirmed cancel transaction enables refund and punish.`